nixos/modules/services/networking/netbird.nix

```
@@ -471,6 +473,17 @@ in
         toClientList (client: optional client.openFirewall client.port)
       );
 
+      # Ports opened on a specific
+      networking.firewall.interfaces = listToAttrs (
+        toClientList (client: {
+          name = client.interface;
+          value.allowedUDPPorts =
+            [ ]
+            ++ optional client.openFirewall 5353 # required for the DNS forwarding/routing to work
```

```suggestion
            ++ optionals client.openFirewall [ 5353 ] # required for the DNS forwarding/routing to work
```

Optional (no pun intended!): use `optionals` to explicitly show that this is a list that is expected.